### PR TITLE
Fixed lifecycle check with long env var list

### PR DESCRIPTION
**Pull request by chaosunsfx**

### CODE REVIEW

File: maestro/lifecycle.py [hunk 1]
```diff
@@ -108,12 +110,21 @@ def _create_env(self):
         return dict((str(k), str(v)) for k, v in env.items())
 
     def _test(self, container=None):
-        return subprocess.call(self.command, env=self._create_env()) == 0
+        env = self._create_env()
+        if self.envfrom == "env":
```
Comment (mpetazzoni) [inline maestro/lifecycle.py]: Use single quotes here too for consistency with the coding style.

File: maestro/lifecycle.py [hunk 2]
```diff
@@ -108,12 +110,21 @@ def _create_env(self):
...
+            return subprocess.call(self.command, env=env) == 0
+        elif self.envfrom == 'stdin':
+            p = subprocess.Popen(self.command, stdin=subprocess.PIPE)
+            p.communicate(json.dumps(env).encode('utf-8'))
+            return p.wait() == 0
+        else:
+            raise ValueError(self.envfrom)
```
Comment (mpetazzoni) [inline maestro/lifecycle.py]: Put a more explicit error message; I was just typing this quickly yesterday.

File: maestro/lifecycle.py (hunk 1 above)
Comment (chaosunsfx) [reply maestro/lifecycle.py]: done

File: maestro/lifecycle.py (hunk 2 above)
Comment (chaosunsfx) [reply maestro/lifecycle.py]: thanks, done

**Pull request merged**